index.html

```diff
@@ -14,9 +14,9 @@
 <meta name="author" content="">
 <title>Boogie</title>
 
-<body>
+<body onLoad="window.scroll(0, 0)">
```

Comment:
idk why, but sometimes when i loaded the page it started at the bottom (like scrolled down). this prevents that.